src/lib/Components/Bidding/Elements/Flex.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components/native"
+import { borders, color, flex, fontSize, space, width } from "styled-system"
+
+/* FIXME: Do not use any */
+export const Flex: any = styled.View`
+  ${borders} ${space} ${width} ${fontSize} ${color} ${flex};
```

Comment:
This actually was written this way as suggested in `styled-system`'s README:

```
${borders}
${space}
${width}
${fontSize}
${color}
${flex}
```

But Prettier made it a one-liner, so we may have to get our tooling right.

Reply:
@yuki24 - semis should do it